lib/internal/modules/esm/hooks.js

```diff
@@ -710,6 +748,7 @@ function pluckHooks({
   globalPreload,
   resolve,
   load,
+  initialize,
```

Comment:
Let’s try to always keep in this order if you don’t mind.

```suggestion
  initialize,
  resolve,
  load,
```